code/game/machinery/emancipation.dm

```diff
@@ -0,0 +1,69 @@
+/obj/machinery/emancipation_grill
+	name = "emancipation grill"
+	desc = "Liberator of luggage. Larcenist of belongings."
+	icon = 'icons/obj/stationobjs.dmi'
+	icon_state = "emancipation_grill"
+	density = FALSE
+	anchored = TRUE
+	use_power = TRUE
+	idle_power_usage = 75
+	active_power_usage = 750
+	flow_flags = IMPASSABLE
+	var/list/obj_whitelist = list() //Things that are okay to go through. Frazzle everything else.
+	var/list/obj_blacklist = list() //Things that aren't okay to go through. Don't frazzle everything else.
+
+/obj/machinery/emancipation_grill/New()
+	..()
+	update_icon()
+
+/obj/machinery/emancipation_grill/power_change()
+	. = ..()
+	update_icon()
+
+/obj/machinery/emancipation_grill/update_icon()
+	if(stat & (BROKEN|NOPOWER))
+		icon_state = initial(icon_state)
+	else
+		icon_state = "[initial(icon_state)]_on"
+
+/obj/machinery/emancipation_grill/proc/emancipate(atom/movable/victim)
+	if(!victim)
+		return
+	if(stat & (BROKEN|NOPOWER))
+		return
+	use_power = 2
```

Comment:
You never set `use_power` back to 1, this means the grill will always use active power after the first time this line is reached. Instead, what I *think* you should do is call `use_power(active_power_usage, power_channel)` here.